flixel/FlxObject.hx

```diff
@@ -548,16 +539,26 @@ class FlxObject extends FlxBasic
 	/**
 	 * Overriding this will force a specific color to be used for debug rect.
 	 */
-	public var debugBoundingBoxColor:Null<Int> = null;
-	
-	public var debugColorScheme(default, null):DebugColorScheme;
+	public var debugBoundingBoxColor(default, set):Null<Int> = null;
+	public var debugBoundingBoxColorNotSolid(default, set):Null<Int> = null;
+	public var debugBoundingBoxColorPartial(default, set):Null<Int> = null;	
```

Comment:
I'm not sure that one should be touched at all. Isn't the use-case for that "use the color I assigned to it no matter what value for `solid` etc is"?